javascript/javascript3/week2/rewriteTimeGetCurrentLocation.js

```diff
@@ -0,0 +1,27 @@
+function getCurrentLocation() {
+  return new Promise((resolve, reject) => {
+    navigator.geolocation.getCurrentPosition((position) => {
+      resolve(position);
+      reject(position);
+    });
+  });
+}
+
+//using promise way
+getCurrentLocation().then((response) => {
+  console.log(response.coords.latitude);
+  console.log(response.coords.longitude);
+});
+getCurrentLocation().catch((response) => console.log(response));
+
+//using asyncway
```

Comment:
Good :)
Try not to write comments in your code. Let your code explain itself instead.
Remember that an async function is also a promise.